config.py

```diff
@@ -1553,6 +1569,9 @@ def get_runs_all_subjects() -> dict:
         valid_runs_subj = get_entity_vals(
             get_bids_root(), entity_key='run',
             ignore_subjects=ignore_subjects)
+        if exclude_runs and subj in exclude_runs:
+            valid_runs_subj = [
+                r for r in valid_runs_subj if r not in exclude_runs[subj]]
```

Comment:
Maybe easier to read:

```suggestion
            valid_runs_subj = [r for r in valid_runs_subj
                               if r not in exclude_runs[subj]]
```